modules/tensor_mechanics/src/materials/crystal_plasticity/ComputeMultipleCrystalPlasticityStress.C

```diff
@@ -77,6 +82,8 @@ ComputeMultipleCrystalPlasticityStress::ComputeMultipleCrystalPlasticityStress(
     _line_search_method(getParam<MooseEnum>("line_search_method").getEnum<LineSearchMethod>()),
     _plastic_deformation_gradient(declareProperty<RankTwoTensor>("fp")),
     _plastic_deformation_gradient_old(getMaterialPropertyOld<RankTwoTensor>("fp")),
+    _thermal_deformation_gradient(declareProperty<RankTwoTensor>("fth")),
+    _thermal_deformation_gradient_old(getMaterialPropertyOld<RankTwoTensor>("fth")),
```

Comment:
Shall we  declare `thermal_deformation_gradient` only when it is coupled to temperature?

Reply:
Thanks @jiangwen84 . Good point. I will make this change.